Bug in make website-assets.tar

Hi @AltGr, since you updated the tutorial with https://github.com/CatalaLang/catala-examples/pull/7, it has not yet been reflected on https://catala-lang.org/. Indeed, before when we had the monorepo for everything I used to do `make website-assets` that covered everything. 

Now there's these instructions here : https://github.com/CatalaLang/catala/blob/master/INSTALL.md#generating-website-assets. However I get the following log : 

```text 
dune build @update-parser-messages --auto-promote
Read 231 sample input sentences and 231 error messages.
dune build @fmt --auto-promote 2>/dev/null
make: [Makefile:138 : format] Erreur 1 (ignorée)
dune build \
	compiler/catala.exe \
	compiler/plugins/ \
	build_system/clerk.exe 
Generating pygment_lexers.ml           
Skipping generation of .mli
Read 231 sample input sentences and 231 error messages.
{ rm -rf catala-examples.tmp && trap "rm -rf catala-examples.tmp" EXIT && git clone https://github.com/CatalaLang/catala-examples --depth 1 --reference-if-able ../catala-examples catala-examples.tmp -b master || git clone https://github.com/CatalaLang/catala-examples --depth 1 --reference-if-able ../catala-examples catala-examples.tmp || git clone -s ../catala-examples catala-examples.tmp -b master || git clone -s ../catala-examples catala-examples.tmp -b master; } && \
make -C catala-examples.tmp \
  CATALA=../_build/default/compiler/catala.exe \
  CLERK=../_build/default/build_system/clerk.exe \
  BUILD=../_build/default \
  ../_build/default/tutorial_en/tutorial_en.html ../_build/default/tutoriel_fr/tutoriel_fr.html ../_build/default/us_tax_code/us_tax_code.html ../_build/default/allocations_familiales/Allocations_familiales.html ../_build/default/allocations_familiales/Allocations_familiales_schema.json ../_build/default/aides_logement/Aides_logement.html ../_build/default/aides_logement/Aides_logement_schema.json
Clonage dans 'catala-examples.tmp'...
remote: Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
make[1] : on entre dans le répertoire « /home/denis/catala/catala-examples.tmp »
../_build/default/compiler/catala.exe html --trace tutorial_en/tutorial_en.catala_en -o ../_build/default/tutorial_en/tutorial_en.html
../_build/default/compiler/catala.exe html --trace tutoriel_fr/tutoriel_fr.catala_fr -o ../_build/default/tutoriel_fr/tutoriel_fr.html
../_build/default/compiler/catala.exe html --trace us_tax_code/us_tax_code.catala_en -o ../_build/default/us_tax_code/us_tax_code.html
../_build/default/compiler/catala.exe html --trace allocations_familiales/Allocations_familiales.catala_fr -o ../_build/default/allocations_familiales/Allocations_familiales.html
../_build/default/compiler/catala.exe json_schema allocations_familiales/Allocations_familiales.catala_fr -o ../_build/default/allocations_familiales/Allocations_familiales_schema.json --scope=InterfaceAllocationsFamiliales
../_build/default/compiler/catala.exe html --trace aides_logement/Aides_logement.catala_fr -o ../_build/default/aides_logement/Aides_logement.html
../_build/default/compiler/catala.exe json_schema aides_logement/Aides_logement.catala_fr -o ../_build/default/aides_logement/Aides_logement_schema.json --scope=CalculetteAidesAuLogementGardeAlternée
make[1] : on quitte le répertoire « /home/denis/catala/catala-examples.tmp »
dune build _build/default/grammar.html _build/default/catala.html _build/default/clerk.html _build/default/catala-examples.tmp/tutorial_en/tutorial_en.html _build/default/catala-examples.tmp/tutoriel_fr/tutoriel_fr.html _build/default/catala-examples.tmp/us_tax_code/us_tax_code.html _build/default/catala-examples.tmp/allocations_familiales/Allocations_familiales.html _build/default/catala-examples.tmp/allocations_familiales/Allocations_familiales_schema.json _build/default/catala-examples.tmp/aides_logement/Aides_logement.html _build/default/catala-examples.tmp/aides_logement/Aides_logement_schema.json grammar.html catala.html clerk.html
Error: Don't know how to build    
_build/default/catala-examples.tmp/tutorial_en/tutorial_en.html
make: *** [Makefile:263 : website-assets-base] Erreur 1
```

Could you debug this build error? Also, could the `website.tar` file be generated by the CI automatically and posted to https://catalalang.github.io/catala/?

--- a/.github/workflows/harness.yml
+++ b/.github/workflows/harness.yml
@@ -137,6 +137,14 @@ jobs:
           cd ~/french-law
           opam --cli=2.1 exec -- make dependencies-ocaml dependencies-js dependencies-python
           opam --cli=2.1 exec -- make bench_ocaml bench_js bench_python
+      - name: Pack website assets
+        run: >-
+          tar cz -hf website-assets.tar.gz
+          -C ~/catala-examples/_build/default/tutorial_en/ tutorial_en.html
+          -C ~/catala-examples/_build/default/tutoriel_fr/ tutoriel_fr.html
+          -C ~/catala-examples/_build/default/us_tax_code/ us_tax_code.html
+          -C ~/catala-examples/_build/default/allocations_familiales/ Allocations_familiales.html Allocations_familiales_schema.json
+          -C ~/catala-examples/_build/default/aides_logement/ Aides_logement.html Aides_logement_schema.json
       - name: Gather all artifacts
         run: |
           cd
@@ -148,6 +156,7 @@ jobs:
           tar cz -hf "artifacts/french-law_ocaml.tar.gz" -C french-law/_build/install/default/lib french-law
           cp catala-examples/_build/french-law_npm.tar.gz artifacts/
           cp catala-examples/_build/french_law_python.tar.gz artifacts/
+          mv website-assets.tar.gz artifacts/
       - name: Upload artifacts
         continue-on-error: true
         # Uploading artifacts works but then return failure with:

--- a/Makefile
+++ b/Makefile
@@ -245,9 +245,11 @@ tests/%: .FORCE
 
 # Note: these are already built by the @doc dune alias
 # (and therefore the doc target here)
-WEBSITE_ASSETS = grammar.html catala.html clerk.html
+WEBSITE_ASSETS_LOCAL_BASE = grammar.html catala.html clerk.html
 
-WEBSITE_ASSETS_EXAMPLES = \
+WEBSITE_ASSETS_LOCAL = $(addprefix _build/default/,$(WEBSITE_ASSETS_BASE))
+
+WEBSITE_ASSETS_EXAMPLES_BASE = \
   tutorial_en/tutorial_en.html \
   tutoriel_fr/tutoriel_fr.html \
   us_tax_code/us_tax_code.html \
@@ -256,19 +258,22 @@ WEBSITE_ASSETS_EXAMPLES = \
   aides_logement/Aides_logement.html \
   aides_logement/Aides_logement_schema.json
 
-WEBSITE_ASSETS_ALL = $(WEBSITE_ASSETS) $(addprefix catala-examples.tmp/,$(WEBSITE_ASSETS_EXAMPLES))
+WEBSITE_ASSETS_EXAMPLES = \
+  $(addprefix _build/default/,$(WEBSITE_ASSETS_EXAMPLES_BASE))
 
-website-assets-base: build
+$(WEBSITE_ASSETS_LOCAL): build
+	dune build $(WEBSITE_ASSETS)
+
+$(WEBSITE_ASSETS_EXAMPLES):
 	$(call local_tmp_clone,catala-examples) && \
 	$(MAKE) -C catala-examples.tmp \
 	  CATALA=../$(CATALA_BIN) \
 	  CLERK=../$(CLERK_BIN) \
 	  BUILD=../_build/default \
-	  $(addprefix ../_build/default/,$(WEBSITE_ASSETS_EXAMPLES))
-	dune build $(addprefix _build/default/,$(WEBSITE_ASSETS_ALL)) $(WEBSITE_ASSETS)
+	  $(addprefix ../,$(WEBSITE_ASSETS_EXAMPLES))
 
-website-assets.tar: website-assets-base
-	tar cf $@ $(foreach file,$(WEBSITE_ASSETS_ALL),-C $(CURDIR)/$(dir _build/default/$(file)) $(notdir $(file)))
+website-assets.tar: $(WEBSITE_ASSETS_LOCAL) $(WEBSITE_ASSETS_EXAMPLES)
+	tar cf $@ $(foreach file,$^,-C $(CURDIR)/$(dir $(file)) $(notdir $(file)))
 
 #> website-assets				: Builds all the assets necessary for the Catala website
 website-assets: website-assets.tar

